### ISSUE TITLE
We broke the needle

### ISSUE THREAD

**Issue by remiolsen** [NationalGenomicsInfrastructure/NGI_dashboards#31]
![screen shot 2016-04-26 at 09 56 03](https://cloud.githubusercontent.com/assets/6576325/14810852/4ae6a3e8-0b95-11e6-8354-585d00908860.png)

For these types of plots I guess, when the number goes over the right-most edge of the box.

**Comment by ewels** [comment]
Hmm. Maybe we could make the text red in that case or something?

**Comment by remiolsen** [comment]
Yeah that could work. Would it turn red if it's in the 0-199 range as well?

Also:
![](http://ct.fra.bz/ol/fz/sw/i55/5/7/12/frabz-ludicrous-speed-GO1-c78084.jpg)
